lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -1069,7 +1075,10 @@ void shouldRemoveOperationalProfileIfMatchingGpg45ProfileIsPresent() throws Exce
                 .thenReturn(clientOAuthSessionItem);
         when(userIdentityService.checkRequiresAdditionalEvidence(any())).thenReturn(false);
         when(userIdentityService.areVcsCorrelated(any())).thenReturn(true);
-        when(configService.enabled(anyString())).thenReturn(false).thenReturn(true);
+        when(configService.enabled(anyString()))
+                .thenReturn(false)
+                .thenReturn(true)
+                .thenReturn(false);
```

Comment:
Is it possible to write this a bit more explicitly? It's quite hard to see what's going on at a glance without just knowing what order the feature flags will be called in any given method. 

It may be that having multiple mock returns based on an input string isn't possible, but I think it would be good to have if we can

Reply:
I believed I tried that earlier and it moaned about it. I think the error it gave was related to stubbing. But I would try doing that again and see if it would work this time